### ISSUE TITLE
Publish 16.02-2.01beta to Maven

### ISSUE THREAD

**Issue by tom91136** [borisbrodski/sevenzipjbinding#13]
Is the `16.02-2.01beta` release ready to be published to maven central? Currently I'm only seeing 9.20-2.00beta, perhaps there is a snapshot repository that I have missed. Would be happy to help out with any blocking issues for 16.02.

**Comment by borisbrodski** [comment]
Hello Tom,

Thank you very much for the readiness to help! Are you skilled in JNI (Java <-> C++?)

You are right, there is no official 16.02-2.01beta release yet. Working on it right now.
You can get different test builds here https://sourceforge.net/projects/sevenzipjbind/files/OldFiles/.

Cheers,
Boris

**Comment by tom91136** [comment]
Good to know that the project is still alive!
I have not used JNI before although I do have experience with JNA but I guess that's not what we need here. I'm fluent in C and Java though so  maybe I can still help with the Java side of things? Perhaps point me to a TODO document and I can do something about those.

I'll try out the old build first and report issues if there are any.

Thanks again for your effort!
Tom

**Comment by borisbrodski** [comment]
Hello Tom,

That's really cool! Thank you very much. I will find something interesting for you :)

Cheers,
Boris

**Comment by optyfr** [comment]
Hello Tom and Boris,
Looked at v16.02-2.01beta, seems to work fine with my [JRomManager](https://github.com/optyfr/JRomManager) project: for example, I was able to use a splitted RAR5 archive to import a mame rom set into many 7z archives using 4 threads at the same time... But since your last post it's still not published to Maven, I wonder what's wrong with that version (compared to latest official 9.20-2.00beta)? I also tried to compile using mingw but then realized that I needed to do same on all possible systems to get each OS/CPU version, and particularly darwin, so I gave up, furthermore my win64 compiled version seems to have failed on all junit tests except only one despite correct build/linking... and by the way, would it even been possible to compile using various crossbuild projects available on github and based on docker? And is rpi's ARM still in project for an eventual 16.02-2.01beta?
Thank you for the work done until then and hope it's not finished
Opty
